SimPEG/Regularization.py

```diff
@@ -801,6 +751,62 @@ def mapping(self, val):
         self._mapping = val
 
 
+###############################################################################
+#                                                                             #
+#              Simple Regularization (no volume contribution)                 #
+#                                                                             #
+###############################################################################
+
+
+class SimpleSmall(BaseRegularization):
+    """
+    Simple Small regularization - L2 regularization on the difference between a
+    model and a reference model. Cell weights may be included. This does not
+    include a volume contribution.
+
+    .. math::
+
+        r(m) = \\frac{1}{2}(\\mathbf{m} - \\mathbf{m_ref})^\top \\mathbf{W}^T
+        \\mathbf{W} (\\mathbf{m} - \\mathbf{m_{ref}})
+
+    where :math:`\\mathbf{m}` is the model, :math:`\\mathbf{m_{ref}}` is a
+    reference model (default Zero) and :math:`\\mathbf{W}` is a weighting
+    matrix (default Identity. If cell weights are provided, then it is
+    :code:`diag(cell_weights)`)
+
+
+    **Optional Inputs**
+
+    :param BaseMesh mesh: SimPEG mesh
+    :param int nP: number of parameters
+    :param IdentityMap mapping: regularization mapping, takes the model from model space to the space you want to regularize in
+    :param numpy.ndarray mref: reference model
+    :param numpy.ndarray indActive: active cell indices for reducing the size of differential operators in the definition of a regularization mesh
+    :param numpy.ndarray cell_weights: cell weights
+
+    """
+
+    _multiplier_pair = 'alpha_s'
+
+    def __init__(self, mesh=None, **kwargs):
+
+        super(SimpleSmall, self).__init__(
+            mesh=mesh, **kwargs
+        )
+
+    @property
+    def W(self):
+        """
+        Weighting matrix
+        """
+        if self.cell_weights is not None:
+            return Utils.sdiag(np.sqrt(self.cell_weights))
```

Comment:
`SimpleSmall` uses the sqrt of the cell weights as the diag if they are included. There is no volume contribution.